oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -685,6 +718,33 @@ private OrchSessionItem createNewOrchSession(String sessionId) {
         return newOrchSessionItem;
     }
 
+    private long getMaxAge(AuthenticationRequest authRequest) {
+        int maxAge;
+        try {
+            maxAge = authRequest.getMaxAge();
+        } catch (Exception e) {
+            LOG.error(
```

Comment:
Do we want this to be a warn instead of an error? Assuming this throwing is RP provided a junk value, in which case we could end up with noisy error alarms

Reply:
RP providing a junk value should be caught earlier on. I can't imagine a case when this exception would be thrown but thought it would be good to add it just in case. Error logging so we're aware if max age isn't functioning as intended

Reply:
Ah makes sense